Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2023 Dynamic Solutions sp. z o.o. sp. k.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use the CMS builder image with cached .m2 repo to create a build artifact.
+# FixMe: use the latest version of the builder image after the CMS release.
+FROM europe-docker.pkg.dev/websight-io/public/websight-cms-builder:1.21.2-test as builder
```

Comment:
This image tag must be changed when `CMS builder` image is released.
Additionally, it should also change when the CMS version is updated (I've added corresponding comment in `pom.xml`).